PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -329,19 +329,51 @@ struct HfCandidateSelectorD0 {
         int pidTrackNegPion = -1;
 
         if (usePidTpcOnly) {
+          /// kaon TPC PID positive daughter
+          selectorKaon.setCustomNSigmaTpcKa(candidate.nSigTpcKa0());
```

Comment:
These can be avoided is one can call `selectorKaon.statusTpc(trackPos, candidate.nSigTpcKa0());`

Reply:
addressed